Makefile

```diff
@@ -188,6 +188,15 @@ githooks-commit:
 githooks-push:
 	@if [ ! -e .git/hooks/pre-push ]; then ln -s ../../bin/pre-push .git/hooks/pre-push; fi
 
+# generate a new shrinkwrap
+shrinkwrap:
+	@rm -rf node_modules
+	@rm npm-shrinkwrap.json
+	@npm install --no-optional
+	@npm install --no-optional # remove this when this is fixed in npm 3
+	@npm install -g shonkwrap
```

Comment:
If somebody has npm global modules in `/usr/local` (e.g. when using [n](https://github.com/tj/n)), this won't work. Running `sudo make shrinkwrap` won't work either because it will create some files as root.